plugins/mx_spook.rb

```diff
@@ -0,0 +1,58 @@
+    "archives", "armada", "armor", "armor-piercing", "arms", "arrangements", "assassinate", "assassination", "assassination", "assault", "atomic bomb", 
+    "bank account", "biological", "blowfish", "bomb", "bomb", "boobytrap", "border", "c4", "camera", "carnivore", "charcoal", "chemical", "child pornography", 
+    "chinese", "class struggle", "claymore", "cocaine", "codebook", "colonel", "commando", "composition b", "conspiracy", "constitution", "cordite", "corporate", 
+    "corrupt", "council", "counter-intelligence", "crack-cocaine", "cracking", "cray", "credit card", "cryptographic", "czar", "d-day", "data haven", 
+    "defcon", "defenses", "democratie", "detcord", "detonate", "detonators", "dictionary", "disruption", "divers", "doctrine", "domestic", "doomsday", 
+    "double agent", "e-bola", "echelon", "efnet", "embassy", "embassy", "embassy", "empire", "encrypt", "enigma", "explosion", "explosive", "faction", 
+    "fertilizer", "fissionable", "flight 800", "freedom", "freemasons", "genetic", "gold bullion", "government", "grenades", "guns", "hack", "harbor", 
+    "heroin", "hijack", "hostage", "hostages", "hydrogen bomb", "illuminati", "impulse", "incendiaries", "infiltration", "infosec", "infrastructure", 
+    "initiators", "insurgent", "intel", "international", "internet worm", "interpol", "jihad", "kamikazi", "kampuchea", "kibo", "kill", "kill", "kill", "kill", 
+    "launch codes", "lead azide", "lead styphante", "liberate", "liberation", "limousine", "lockpick", "loyalist", "main charge", "marijuana", "martyr", 
+    "maverick", "mercury fulminate", "microfiche", "microfilm", "minefield", "mines", "motorcade", "motorola", "mouvement", "munitions", "napalm", "nationalist", 
+    "nitric acid", "nitrocellulose", "nuclear", "oppressed", "orthodox", "password", "picric acid", "pipe-bomb", "plague", "platter charge", "plutonium", 
+    "plutonium", "policy", "political", "pre-teen", "president", "president", "primers", "private key", "propaganda", "psyops", "public key", 
+    "pulse detonation engine", "radar", "rail gun", "rebel", "remailer", "resistance", "revolucionario", "rijndael", "robotic", "rockets", "root-servers.net", 
+    "rubella", "salt peter", "sanctions", "satelliate", "satellite", "satellite phone", "secret", "secret key", "secret service", "secure", "security", 
+    "sequence", "shaped charge", "smallpox", "smuggle", "sniper", "sniper", "socialist", "space station", "spy", "steganography", "strategic", "submarine",
+    "subsonic", "suicide", "suicide bombing", "sulfur", "supercomputer", "supersonic", "surveillance", "teflon bullets", "terminate", "terrorist", 
+    "theater missile defense", "thermite", "timers", "tunneling", "undercover", "undernet", "uranium", "virus", "warfare", "warrant", "weapons", 
+    "white noise generator", "wiretap", "zenith"]
+  
+  listen_to :channel
+  def listen(m)
+    match = /M-x spook\b/.match(m.message)
```

Comment:
ehhhhhh but how expensive is "pretty expensive", really? steggybot listens to "who is" and seems to be not dying, anyhow...